### ISSUE TITLE
Create a password change ticket: "mark_email_as_verified" parameter missing.

### ISSUE THREAD

**Issue by debied** [auth0/auth0-PHP#522]
### Describe the problem

According to the API documentation https://auth0.com/docs/api/management/v2#!/Tickets/post_password_change and other docs/guides (i.e. https://auth0.com/docs/email/send-email-invitations-for-application-signup#create-password-change-tickets) there's the parameter "mark_email_as_verified" available when creating password change tickets.

However this parameter isn't available in your SDK:
https://github.com/auth0/auth0-PHP/blob/44caed3aa72f9c9b60d9b7de80dd1909e7e62281/src/API/Management/Tickets.php#L127

```php
    /**
     *
     * @param null|string  $user_id         User for whom the ticket should be created. Conflicts with: $connection_id, $email
     * @param null|string  $email           Email address of the user for whom the ticket should be created. Requires $connection_id. Conflicts with $user_id.
     * @param null|string  $new_password    New password to assign to the user.
     * @param null|string  $result_url      URL the user will be redirected to in the classic Universal Login experience once the ticket is used.
     * @param null|string  $connection_id   Connection to use, allowing user to be specified using $email, rather than $user_id. Requires $email. Conflicts with $user_id.
     * @param null|integer $ttl             Number of seconds this ticket will be valid before expiring. Defaults to 432000 seconds (5 days.)
     * @param null|string  $client_id       If provided for tenants using New Universal Login experience, the user will be prompted to redirect to the default login route of the corresponding application once the ticket is used.
     * @param null|string  $organization_id If provided, the organization_id and organization_name will be included as query arguments in the link back to the application.
     *
     * @return mixed
     */
    public function createPasswordChangeTicketRaw(
        $user_id = null,
        $email = null,
        $new_password = null,
        $result_url = null,
        $connection_id = null,
        $ttl = null,
        $client_id = null,
        $organization_id = null
    )
```

### What was the expected behavior?

The parameter "mark_email_as_verified" should be available when using the SDK to create a "Password change ticket".

### Reproduction

Open the API documentation and compare the docs with the PHP SDK implementation.

### Environment

- **Version of this library used:** ^7.9
- **Which framework are you using, if applicable:** Symfony
- **Other modules/plugins/libraries that might be involved:** None
- **Any other relevant information you think would be useful:** None

**Comment by evansims** [comment]
Hey, @debied 👋 Thanks for the reporting this

`Stable/7.x` I've created [a pull request](https://github.com/auth0/auth0-PHP/pull/523), which will add these arguments to the relevant endpoint methods and pass the parameters with the API request.
`Beta/8.x` N/A

**Comment by evansims** [comment]
Hey again @debied 👋  Just to let you know, this change has been merged. I'm off until next week, so I'll get a new release shipped with the change when I return.

**Comment by debied** [comment]
Thanks @evansims for your support and the new release.

**Comment by debied** [comment]
Hey again @evansims 
Can you tell me when the fix will be released? Thanks for your support.

**Comment by evansims** [comment]
Hey @debied 👋  Just released with 7.9.2!
